test/lens.spec.js

```diff
@@ -42,24 +42,21 @@ describe('Lens Functions', () => {
       expect(l.get()).to.equal(1)
       l.set(5)
       expect(l.get()).to.equal(5)
-    })
-    it('lensProp deep', () => {
-      let l = F.lensProp('x.a', {
+      //lensProp deep
+      let ld = F.lensProp('x.a', {
         x: {
           a: 1,
         },
       })
-      expect(l.get()).to.equal(1)
-      l.set(5)
-      expect(l.get()).to.equal(5)
+      expect(ld.get()).to.equal(1)
+      ld.set(5)
+      expect(ld.get()).to.equal(5)
     })
     it('lensOf', () => {
-      let l = F.lensOf({
-        a: 1,
-      })
-      expect(l.a.get()).to.equal(1)
-      l.a.set(5)
-      expect(l.a.get()).to.equal(5)
+      let object = { a: 1 }
+      let lens = F.lensOf(object)
+      lens.a.set(8)
+      expect(lens.a.get()).to.deep.equal(8)
```

Comment:
Deep equal is for objects with more than one level of nesting. It's not necessary for primitives.

Reply:
Thank you @stellarhoof. I fixed it.